lib/order.rb

```diff
@@ -0,0 +1,38 @@
+require_relative 'takeaway'
+class Order < Menu
+  
+  attr_accessor :dishes, :my_choices, :cart, :menu
+  def initialize
+  @my_order = {}
+  @cart = []
+  end
+  
+
+  def add_dish_to_order(dish, quantity)
+    @my_order = {dish => quantity }
+    @add_to_cart
+  end
+
```

Comment:
nice use of SRP to keep each method responsible for a single thing, and nice short clear code.